fenix/app/src/main/res/values/strings.xml

```diff
@@ -377,7 +377,8 @@
     <string name="preferences_https_only_title">HTTPS-Only Mode</string>
 
     <!-- Preference for removing cookie/consent banners from sites automatically. See reduce_cookie_banner_summary for additional context. -->
-    <string name="preferences_cookie_banner_reduction">Cookie Banner Reduction</string>
+    <string name="preferences_cookie_banner_reduction" moz:RemovedIn="114" tools:ignore="UnusedResources">Cookie Banner Reduction</string>
+    <string name="preferences_cookie_banner_reduction_2">Cookie banner reduction</string>
```

Comment:
Since this string is only changing capitalization, we don't need to update the string ID here, as localizers chose capitalization depending on their locale

Reply:
Done